apps/web/src/app/domain/pages/dark-mode/dark-mode.page.html

```
@@ -7,87 +7,444 @@
+    </div>
+
+    <div class="overflow-hidden transition-all duration-200 ease-in-out">
+      <nav
+        class="text-muted-foreground mb-3 flex gap-6"
+        role="tablist"
+        aria-label="Installation method"
+        #tabList
+        (keydown.{arrowleft,arrowright,home,end,space,enter}.prevent)="onInstallKeyDown($event, tabList)"
+      >
+        <button
+          id="cli-tab"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)